mindsdb/integrations/handlers/discord_handler/discord_tables.py

```diff
@@ -0,0 +1,188 @@
+import pandas as pd
+from typing import Text, List, Dict, Any
+
+from mindsdb_sql.parser import ast
+
+from mindsdb.integrations.libs.api_handler import APITable
+from mindsdb.integrations.utilities.sql_utils import extract_comparison_conditions
+
+from mindsdb.integrations.libs.response import HandlerResponse as Response
+
+from mindsdb.integrations.handlers.utilities.query_utilities.insert_query_utilities import (
+    INSERTQueryParser
+)
+
+from mindsdb.utilities.log import get_log
+
+logger = get_log("integrations.discord_handler")
+
+class DiscordTable(APITable):
```

Comment:
Let's call this `MessagesTable` instead, just in case we need to add other tables in the future. You will need to make some changes within the handler, in registering the table as well once this is done.